tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
+[jaeger](https://github.com/uber/jaeger-client-go) that allows you to
+instrument operations in your application.
+Jaeger tracer can be configured with an optional logger that logs errors/spans
+and a stats reporter for emitting metrics.
+Note that the tracer is initialized by default with usage of UberFx modules.
+If you decide to use it standalone, read on for an example on how to initialize
+the tracer.
+
+## Sample usage
+```go
+package main
+
+import (
+  "go.uber.org/fx/tracing"
+  "go.uber.org/fx/ulog"
+
+	"github.com/uber/jaeger-client-go/config"
+)
+
+func main() {
+  logger := ulog.Logger()
+  statsReporter := // initialize stats reporter
```

Comment:
Creating a statsReporter is not straightforward right now and even for a Nop implementation, we have our own struct. But this should be replaced with tally's soon. I didn't want to add confusion with that here.